logstash-core/src/main/java/org/logstash/ackedqueue/Queue.java

```diff
@@ -139,6 +139,10 @@ public long getCurrentByteSize() {
         return this.currentByteSize;
     }
 
+    public long getCurrentPhysicallyPersistedByteSize() {
```

Comment:
+1. maybe since it seems we are currently pretty much always using "bytes" in the names, I'd probably keep it for consistency.